src/mmw/js/src/data_catalog/templates/contributeModal.html

```diff
@@ -0,0 +1,51 @@
+<div class="modal-dialog contribute-modal" role="document">
+    <div class="modal-content">
+        <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+            </button>
+            <h2 class="modal-title">Contribute Data</h2>
+        </div>
+        <div class="modal-body">
+            <p>
+                Add your own datasets to the catalogs below to share them with
+                a wide audience. Datasets added there are included in searches
+                performed in Monitor My Watershed.
```

Comment:
Do we need the ® here after MMW?

Reply:
Potentially. Going to leave this open until we have final copy from Stroud. Thanks for taking a look!